src/apprt/gtk/Window.zig

```diff
@@ -167,7 +167,7 @@ pub fn init(self: *Window, app: *App) !void {
         // If we're using an AdwWindow then we can support the tab overview.
         if (self.tab_overview) |tab_overview| {
             if (comptime !adwaita.versionAtLeast(1, 4, 0)) unreachable;
-            assert(self.isAdwWindow());
+            assert(self.app.config.@"gtk-adwaita" and adwaita.versionAtLeast(1, 3, 0));
```

Comment:
Is there a reason that the version numbers are different?

Reply:
Fixed